<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackupPage</class>
 <widget class="QWizardPage" name="BackupPage">
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QProgressBar" name="m_progressBar">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>9714</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_status">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
